Source/WebCore/dom/ContainerNode.cpp

```diff
@@ -592,14 +594,14 @@ ExceptionOr<void> ContainerNode::replaceChild(Node& newChild, Node& oldChild)
     // If it is, it can be deleted as a side effect of sending mutation events.
     ASSERT(refCount() || parentOrShadowHostNode());
 
-    Ref<ContainerNode> protectedThis(*this);
+    Ref protectedThis { *this };
```

Comment:
This seems like the old idiom. In future we rely on the caller having protected `this`. But I guess it will always be too risky to remove this old protect code?

Reply:
Correct, I am keeping those for now but this is the old idiom and it should not be needed anymore once our code has been fully ported to newer idioms. However, I think it is too early to remove those as our code is far from being fully ported.